consensus/types/src/chain_spec.rs

```diff
@@ -108,6 +108,8 @@ pub struct ChainSpec {
      */
     pub eth1_follow_distance: u64,
     pub seconds_per_eth1_block: u64,
+    pub deposit_chain_id: u64,
```

Comment:
We are currently checking only `network_id` in the eth1 crate to make sure beacon node is connected to the correct eth1 chain. I think we should check `chain_id` as well to ensure that we aren't reading any bogus logs from an eth1 chain with a different chain id.

Reply:
Good call, I made an issue here: https://github.com/sigp/lighthouse/issues/1815